README.md

```diff
@@ -31,7 +31,7 @@ Select all that apply:
 Explain what is a REPL, and why is it important for us as developers and help with debugging?
 
 ```text
-
+It is a site for checking and printing JavaScript code, and it is important for us to get in the habit of troubleshooting in programing languages because there will be constant bugs.
```

Comment:
REPL, stands for "Read-Eval-Print Loop". It's an environment that allows us to run JavaScript code one line at a time. It's especially useful in helping us to experiment and debug our code, testing portions of our code before continuing.

Important to note that other languages can have REPLs